src/views/AdminEmailAdd.vue

```diff
@@ -0,0 +1,55 @@
+<script setup>
+    import { ref, onMounted } from 'vue';
+    import router from '../router';
+import emailMessageServices from "../services/emailMessageServices";
+
+    let emailName = ref('');
+    let emailDesc = ref('');
+    let emailText = ref('');
+
+   
+    function save(){
+        let emailData = {
+            name: emailName.value,   
+            description: emailDesc.value,
+            text: emailText.value,
+        };
+
+        emailMessageServices.create(emailData);
+        router.push({ name: 'adminEmailManage'});
+    }
+    function cancel(){
+        router.push({ name: 'adminEmailManage'});
+    }
+</script>
+
+<template>
+    <div>
+        <v-title class="text-h5" style="font-weight: bold;">Add Email Message</v-title>
+    </div>
+
+    <div class="pa-4">
+        
+        <v-card style="background-color:#D5DFE7" class="pa-4">
+           
+            <div>
+                <p class="pl-5" style="font-weight: bold;">Name</p>
+                <v-text-field class="pl-5 pr-5" label="enter name for message" v-model="emailName"></v-text-field>
+            </div>
+            <div>
+                <p class="pl-5" style="font-weight: bold;">Description</p>
+                <v-text-field class="pl-5 pr-5" label="enter description" v-model="emailDesc"></v-text-field>
+            </div>
+            <div>
+                <p class="pl-5" style="font-weight: bold;">Content</p>
+                <v-text-field class="pl-5 pr-5" label="enter the content of the email" v-model="emailText"></v-text-field>
```

Comment:
- Change labels to remove "enter" verbage (ie. "enter description"
- If the labels are redundant with the <p> tags above the fields, change the labels to provide more useful information, or remove them to be consistent